tests/e2e/mnist_raycluster_sdk_test.py

```diff
@@ -0,0 +1,171 @@
+from kubernetes import client, config
+import kubernetes.client
+
+import os
+
+from time import sleep
+
+import ray
+
+from torchx.specs.api import AppState, is_terminal
+
+from codeflare_sdk.cluster.cluster import Cluster, ClusterConfiguration
+from codeflare_sdk.job.jobs import DDPJobDefinition
+
+import pytest
+
+from support import random_choice, get_ray_image
+
+# Creates a Ray cluster, and trains the MNIST dataset using the CodeFlare SDK.
+# Asserts creation of AppWrapper, RayCluster, and successful completion of the training job.
+# Covers successfull installation of CodeFlare-SDK
+
+
+class TestMNISTRayClusterSDK:
+    def setup_method(self):
+        # Load the kube config from the environment or Kube config file.
+        config.load_kube_config()
+
+        # Initialize Kubernetes client
+        self.api_instance = client.CoreV1Api()
+        self.custom_api = kubernetes.client.CustomObjectsApi(
+            self.api_instance.api_client
+        )
+
+    def teardown_method(self):
+        if hasattr(self, "namespace"):
+            self.api_instance.delete_namespace(self.namespace)
+        if hasattr(self, "configmap"):
+            self.api_instance.delete_namespaced_config_map(
+                self.configmap.metadata.name, self.namespace
+            )
```

Comment:
What ConfigMap is deleted here? I didn't find any other reference for `configmap` attribute.

Reply:
Good catch, forgot to remove it, we no longer need to create one - Removed